README.md

```diff
@@ -149,12 +173,22 @@ end
 4. Run rubocop lint (`bundle exec rubocop -R --format simple`)
 5. Run rspec tests (`bundle exec rspec`)
 6. Push your branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
+7. Create a new Pull Request to `development` branch
 
```

Comment:
What do you think about to add a releases link in Contributing paragraph?
We can add a link to releases page, and there list all changes in gem 
📢 [See what's changed in a recent version](https://github.com/Wolox/wor-paginate/releases)